.github/workflows/tasklist-e2e-tests.yml

```diff
@@ -3,27 +3,27 @@ name: Tasklist E2E Tests
 on:
   push:
     branches:
-      - 'main'
```

Comment:
is the change from single to double quotes intentional ? or made by a formatting linter ?

Reply:
@houssain-barouni made by linter